src/imp/tracer_imp.js

```diff
@@ -245,6 +245,8 @@ export default class Tracer extends opentracing.Tracer {
         this.addOption('delay_initial_report_millis', { type: 'int', defaultValue: 1000 });
         this.addOption('error_throttle_millis', { type: 'int',     defaultValue: 60000 });
         this.addOption('logger',                { type: 'function', defaultValue: this._printToConsole.bind(this) });
+        this.addOption('flush_span_buffer_consecutive_errors',
```

Comment:
This is just semantic but in most of our tracers _flush_ means to try and send a report to the satellites. So here we should call it `clear_span_buffer_on_consecutive_errors`